src/Libraries/Microsoft.Extensions.Diagnostics.ExceptionSummarization/README.md

```diff
@@ -1 +1,77 @@
-README
+# Microsoft.Extensions.Diagnostics.ExceptionSummarization
+
+This provides the ability to extract essential information from well-known exception types and return a single string that can be used to create low-cardinality diagnostic messages for use in telemetry.
+
+## Install the package
+
+From the command-line:
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.Diagnostics.ExceptionSummarization
+```
+
+Or directly in the C# project file:
```

Comment:
It could be VB or F#

Reply:
We already have this pre-defined for all readmes. Let's do a separate global change once we agree on this, but keep it consistent for now a least.